packages/react-effect/README.md

```diff
@@ -0,0 +1,121 @@
+# `@shopify/react-effect`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-effect.svg)](https://badge.fury.io/js/%40shopify%2Freact-effect.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-effect.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-effect.svg)
+
+A component and set of utilities for performing effects within a universal React app. This package gives you the tools you need to extract details from an application during server rendering using only a single pass.
```

Comment:
I don't think you need give the example of extracting details. Could be combined to form the following one-liner. 

>This package contains a component and set of utilities for performing multiple effects during one single pass of server rendering in a universal React application.